web/toDoList.jsp

```diff
@@ -1,160 +1,147 @@
+            <%
+                for (ToDoItem item : toDoListItems) {
+            %>
+            <div id="listItemId-<%= item.ID %>" class="toDoListDataStyle">
                 <span id="item-<%= item.ID %>" class="itemStyle " contenteditable="true"><%= item.message %>
                 </span>
                 <span class="checkboxStyle checkbox-primary" id="checkboxId-<%= item.ID %>"><input class="checkboxSize"
                                                                                                    type="checkbox"/></span>
+                <%if (item.timer.equals("")) {%>
                 <span id="remainderButton-<%= item.ID %>" class="remainderButtonStyle"
-                      value="Remainder"><span class="glyphicon glyphicon-time"></span>
-                    <span id="displayTimeId-<%= item.ID %>" style="display : none"></span>
-                </span>
-                </div>
-                <div id="panel-<%= item.ID %>" class="panelSlide">
-                    <div class="timerStyle ">
-                        <div class="col-sm-5 input-group date datetimepicker">
-                            <input id="dateId-<%= item.ID %>" type="text" class="form-control"/>
-                            <div class="input-group-addon glyphicon glyphicon-calendar"></div>
-                        </div>
-                    </div>
-                    <div class="timerButtons ">
-                        <div id="cancelButton-<%= item.ID %>" class="col-sm-2 btn btn-default">Cancel</div>
-                        <div id="timeSetButton-<%= item.ID %>" class="col-sm-2 btn btn-default">Done</div>
-                    </div>
-                </div>
-                <%
-                        lastMessageId = item.ID;
-                    }
-                %>
+                      value="Remainder"><span class="glyphicon glyphicon-time"></span></span>
+                <% } else {%>
+                <span class="timeLeftMessageStyle" id="displayTimeId-<%= item.ID %>" style="cursor: pointer"></span>
+                <script> getUserTimeData("<%=item.timer %>", <%= item.ID %>);
+                </script>
+                <% } %>
             </div>
-            <div class="inputTextOuter">
-                <div class="col-sm-5 inputTextInner">
-                    <input type="text" class="form-control" id="todoItemName"
-                           placeholder="Your To-Do goes here.."/>
+            <%
+                }
+            %>
+        </div>
+        <div id="panel" class="panelSlide">
+            <div class="timerStyle ">
+                <div class="col-sm-5 input-group date datetimepicker">
+                    <input id="dateId" type="text" class="form-control"/>
+                    <div class="input-group-addon glyphicon glyphicon-calendar"></div>
                 </div>
             </div>
+            <div class="timerButtons ">
+                <div id="cancelButton" class="col-sm-2 btn btn-default">Cancel</div>
```

Comment:
instead of btn-default, use btn-link.  negative buttons should usually look less prominent. otherwise, users will get confused what button to click.